functions/inputs/storage/storage.go

```diff
@@ -20,6 +20,7 @@ type HostLookup interface {
 
 type BucketLookup interface {
 	Lookup(orgID platform.ID, name string) (platform.ID, bool)
+	FindAllBuckets(orgID platform.ID) ([]*platform.Bucket, int)
```

Comment:
What purpose does the int serve?

Reply:
the underlying service interface returns a count of the number of buckets, so I pass it up.  it's not essential.